### DISCUSSION TITLE
Welcome to lichtblick Discussions!

### DISCUSSION THREAD

**Discussion by laisspportugal** [lichtblick-suite/lichtblick#112] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Woodii1998** [comment]
@laisspportugal Currently, this project is the most promising one after Foxglove abandoned open source. However, it still has a significant gap compared to Foxglove Studio in terms of functionality and code maintenance (such as h264 support, major version upgrades of eslint, etc.). I would like to ask about your future plans for this project. Is there a roadmap that can be made public, which would facilitate the open source community's involvement in contributing to the development of this project?

cc: @luluiz @aneuwald-ctw

**Comment by jdsika** [comment]
Hi @Woodii1998 
we are continously working on a clear process and communication regarding the involvment of the broader community - the **we** is still in need to be defined and how **you** can be part of it.  This includes the clear guideleines how we as a community can steer this project and how you could start leading it or be its steward. The community can do work on PRs here any time - I personally think that people might still feel a little shy but they should not :)
There is no intention to publish something like an official roadmap as this might have as a prerequisite that multiple parties sit together in community calls writing down expectations. The work on such an open source project is absolutely voluntary which means that the commitment to such goals would be voluntary as well - from each individual. 
This way I would say I welcome your suggetions and contributions here and I am happy for your first step by replying to this post!
